extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftProcessor.java

```diff
@@ -148,32 +147,18 @@ public void openshiftRequirementsJvm(OpenshiftConfig openshiftConfig,
             String jarDirectory = config.jarDirectory
                     .orElse(baseImage.map(i -> i.getJarDirectory()).orElse(config.FALLBACK_JAR_DIRECTORY));
             String pathToJar = concatUnixPaths(jarDirectory, jarFileName);
-            String classpath = appDeps.stream()
-                    .map(d -> d.getArtifact().getGroupId() + "." + d.getArtifact().getPath().getFileName())
-                    .map(s -> concatUnixPaths(jarDirectory, "lib", s))
-                    .collect(Collectors.joining(":"));
 
             // If the image is known, we can define env vars for classpath, jar, lib etc.
             baseImage.ifPresent(b -> {
                 envProducer.produce(KubernetesEnvBuildItem.createSimpleVar(b.getJarEnvVar(), pathToJar, null));
-                if (libRequired) {
-                    envProducer.produce(KubernetesEnvBuildItem.createSimpleVar(b.getJarLibEnvVar(),
-                            concatUnixPaths(jarDirectory, "lib"), null));
-                    envProducer.produce(KubernetesEnvBuildItem.createSimpleVar(b.getClasspathEnvVar(), classpath, null));
-                }
```

Comment:
@iocanel I was not 100% sure this block can be removed because I have no idea whether those env vars are used for something else beyond setting classpath. Could you please confirm?

Reply:
I can add it back if necessary.

Reply:
Given that the classpath is meant to be determined from the jar metadata, I think it safe to also remove environment variables.